src/main/webapp/game.js

```diff
@@ -10,11 +13,52 @@ const CSS_MESSAGE_CLASS_DICT = {
 };
 // TODO: @salilnadkarni, replace with userid from cookie (in datastore)
 const USER_ID = "_" + Math.random().toString(36).substr(2, 9);
```

Comment:
This const is legacy code, so probably best to remove it.

Reply:
do you still need the todo @salnad

Reply:
No I don't